content/node-runners/setup/linux.md

```diff
@@ -156,3 +155,43 @@ After running the `restart` command make sure to check the service health and cu
 using previously mentioned commands.
 
 If node is refusing to restart and load the changes you've made, you can try to restart systemd daemon itself by running `systemctl daemon-reexec`
+
+## Install development (snapshot) version of the node
```

Comment:
We already mention something similar here: https://github.com/mysteriumnetwork/documentation/blob/master/content/node-runners/setup/linux.md#latest-non-stable-release

We could just extend it a bit more there, explaining that we have `pre-` and `dev` builds and I think that would be just fine.

Reply:
That section does not explain how to install it, plus it doesn't mention anything about our snapshots. It needs a separate section and it's nice to have it detailed enough that I would be able to just copy paste stuff and it would work.